spec/unit/optmath_spec.lua

```diff
@@ -0,0 +1,49 @@
+describe("Math module", function()
+
+    setup(function()
+        require("commonrequire")
+        Math = require("optmath")
+    end)
+
+    it("should round away from zero", function()
+        local num = 1.5
```

Comment:
I thought it looked clearer. Or do you mean as opposed to a "global" local at the top? That'd be a lack of conscious reflection after writing and duplicating the first test.